fs_brokers/apache_hdfs_broker/build.sh

```diff
@@ -42,6 +42,12 @@ rm -rf ${BROKER_OUTPUT}
 install -d ${BROKER_OUTPUT}/bin ${BROKER_OUTPUT}/conf \
            ${BROKER_OUTPUT}lib/
 
+# download aliyun hadoop oss jar, this can not be found in maven repository, so we download it manually
+wget http://gosspublic.alicdn.com/hadoop-spark/hadoop-oss-2.7.2.tar.gz
+tar xzf hadoop-oss-2.7.2.tar.gz
+mv hadoop-oss-2.7.2/hadoop-aliyun-2.7.2.jar ${BROKER_OUTPUT}/lib/
+rm -r hadoop-oss-2.7.2 hadoop-oss-2.7.2.tar.gz
```

Comment:
Is it must be 2.7.2?
Does this not work?
`https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aliyun/2.9.2`

Reply:
Not work, I have tried the 2+ and 3+ version of lib from maven repository. So we have to download it by hand.

Reply:
Hadoop-aliyun(OSS) was merged into hadoop since  hadoop 2.9.1